src/OneSignal.ts

```diff
@@ -471,7 +487,7 @@ export default class OneSignal {
     if (emailProfile.emailId) {
       await OneSignalApi.updatePlayer(appId, emailProfile.emailId, {
         tags: tags,
-        email_auth_hash: emailProfile.emailAuthHash,
+        identifier_auth_hash: emailProfile.identifierAuthHash,
```

Comment:
Just noting here, the OneSignal backend needs to be deployed before this PR is due to this change